lib/adapters/idb/idb.js

```diff
@@ -1039,8 +1039,7 @@ function init(api, opts, callback) {
             };
           };
         }).catch(function (err) {
-          api._blobSupport = false;
-          checkSetupComplete();
+          return false; // error, so assume unsupported
```

Comment:
I'm not sure why we had this; it could potentially call `checkSetupComplete()` twice, which is wrong.